docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/fractribution/index.md

```diff
@@ -38,6 +39,7 @@ All variables in Snowplow packages start with `snowplow__` but we have removed t
 | Variable Name                    | Description                                                                                                                                                                          | Default   |
 | -------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------- |
 | `channels_to_exclude`            | List of channels to exclude from analysis (empty to keep all channels). For example, users may want to exclude the 'Direct' channel from the analysis.                               | `[]`      |
+| `channels_to_include`            | List of channels to include in the analysis (empty to keep all channels). For example, users may want to include the 'Direct' channel only in the analysis.                               | `[]`      |
```

Comment:
Add to the json thingy please https://www.notion.so/keep-in-the-snow/Documentation-Updates-9d1822916caa414d9f1bb931a462f1f9?pvs=4